internal/plugins/helm/v1/scaffolds/internal/templates/makefile.go

```diff
@@ -70,42 +70,51 @@ IMG ?= {{ .Image }}
 
 all: docker-build
 
-# Run against the configured Kubernetes cluster in ~/.kube/config
-run: helm-operator
+##@ Targets
```

Comment:
Same as above, align with the corresponding kb PR.

Reply:
This PR was not merged yet. 
we did not need to do it now.

Reply:
I think you shouldn't add help here, and wait for upstream kb adds it. I think that would be